src/main/java/seedu/address/logic/parser/Flag.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.parser;
+
+/**
+ * A flag that marks the beginning of a type that the command is applied to.
+ * E.g. '-p' in 'delete -p 1'.
+ */
+public class Flag {
```

Comment:
Great abstraction of the flag class. Will be useful for my training command as well.